src/com/farzaneh/model/Person.java

```diff
@@ -0,0 +1,74 @@
+package com.farzaneh.model;
+
+public class Person {
```

Comment:
I recommend you to overrid hashCode and equals based on personId

Reply:
done